internal/tools/parameters.go

```diff
@@ -238,19 +257,20 @@ func (ps Parameters) Manifest() []ParameterManifest {
 
 // ParameterManifest represents parameters when served as part of a ToolManifest.
 type ParameterManifest struct {
-	Name        string             `json:"name"`
-	Type        string             `json:"type"`
-	Description string             `json:"description"`
-	AuthSources []string           `json:"authSources"`
-	Items       *ParameterManifest `json:"items,omitempty"`
+	Name         string             `json:"name"`
+	Type         string             `json:"type"`
+	Description  string             `json:"description"`
+	AuthServices []string           `json:"authServices"`
```

Comment:
The change in the protocol here means that we'll need to update the SDKs as well -- @anubhav756 @twishabansal FYI